src/main/java/net/earthcomputer/clientcommands/command/CTimeCommand.java

```diff
@@ -1,13 +1,19 @@
 package net.earthcomputer.clientcommands.command;
 
 import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.IntegerArgumentType;
+
+import net.earthcomputer.clientcommands.features.ClientTimeModifier;
 import net.fabricmc.fabric.api.client.command.v1.FabricClientCommandSource;
 import net.minecraft.SharedConstants;
 import net.minecraft.client.MinecraftClient;
+import net.minecraft.command.argument.TimeArgumentType;
 import net.minecraft.text.TranslatableText;
 
+import static com.mojang.brigadier.arguments.IntegerArgumentType.getInteger;
 import static net.earthcomputer.clientcommands.command.ClientCommandHelper.*;
 import static net.fabricmc.fabric.api.client.command.v1.ClientCommandManager.*;
+import static net.minecraft.command.argument.TimeArgumentType.time;
```

Comment:
Prefer using [`CTimeArgument`](https://github.com/xpple/clientarguments/blob/1ade89e78e90aecf4c4a443eaedd0070ab41ba1c/src/main/java/dev/xpple/clientarguments/arguments/CTimeArgumentType.java) to be consistent with other non-Brigadier argument types.